addons/mail/static/src/models/messaging.js

```diff
@@ -398,6 +398,9 @@ registerPatch({
             default: {},
             isCausal: true,
         }),
+        notificationAlertView: one('NotificationAlertView', {
+            default: {},
+        }),
```

Comment:
This is not the correct place to put this.
Current PR is making the assumption there is only one `NotificationAlertView` ever displayed at the same time, but there is absolutely no guarantee of that.

Only the usage of the container is controlling how many are potentially existing, if at all.

Container should follow similar shape of other containers and manually handle creating and deleting of associated view record.